decentraland/unity-renderer
# style: linked wearables dropdown

## What does this PR change?
The linked wearables dropdown was in the backpack sections hierarchy instead with the wearables categories which is the place it belongs.

BEFORE
<img width="1335" alt="Screen Shot 2022-05-25 at 20 05 09" src="https://user-images.githubusercontent.com/51088292/170385372-cebb213a-e7e8-4442-8125-bf2422e98285.png">

AFTER
<img width="1335" alt="Screen Shot 2022-05-25 at 20 18 28" src="https://user-images.githubusercontent.com/51088292/170385385-6c584f73-7a2f-40fd-be87-3b48a3e328e7.png">


## How to test the changes?
1. Go to: https://play.decentraland.zone/?renderer-branch=style/LinkedWearablesDropdown
2. Open the avatar section in the backpack. You will notice that the linked wearables dropdown has been moved on the wearables categories hierarchy.
